client/src/App.js

```diff
@@ -75,7 +76,8 @@ const App = () => {
             path="/login"
             render={(routerProps) => <Login setAuthenticated={setAuthenticated} {...routerProps} />}
           />
-          <Route path="/profiles/:id" component={ProfilePage}/>
+          <Route path="/profiles/:id" component={ProfilePage} />
+          <Route path="/projects/add/:id" component={AddProject} />
```

Comment:
I'm curious as to why the projects/add/id format for the route. 
I would have expected something like users/id/projects, but don't know if I'm maybe missing something

Reply:
The route is subject to change. It's probably better to do something like users/:id/projects/add or something like that

Reply:
Maybe. It would definitely be more intuitive. For instance, at first glance I had no idea what the :id actually referred to in the path. Could maybe even just do "/projects" because the project is created by the current user?